tests/openapi/test_snapshot.py

```
@@ -370,15 +370,15 @@ def test_full_snapshot_security():
     snapshot_name = "/etc/passwd"
     response = requests.get(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)